openrtb/bid.go

```diff
@@ -43,3 +43,8 @@ func (bid Bid) Validate() error {
 
 	return nil
 }
+
+func (b *Bid) Copy() *Bid {
```

Comment:
Why do we need pointer method receiver?

Reply:
I guess we don't need it here since it's just a copy.